packages/block-library/src/site-logo/editor.scss

```diff
@@ -80,3 +80,23 @@
 		}
 	}
 }
+.editor-styles-wrapper {
```

Comment:
Is this parent selector needed?

Reply:
It's needed as there are some styles with higher specificity involving `.editor-styles-wrapper` and paragraphs.

Without it:
<img width="621" alt="Screenshot 2021-06-25 at 4 29 51 PM" src="https://user-images.githubusercontent.com/16275880/123432101-af8c7b80-d5d2-11eb-959d-7684c3a2e18b.png">